framework/libra-framework/sources/ol_sources/vouch.spec.move

```diff
@@ -0,0 +1,93 @@
+spec ol_framework::vouch {

```

Comment:
Would be great if you added the same code comments as you did in burn.move.spec: preconditions, aborts, postconditions.